src/logic/Preferences.swift

```diff
@@ -417,6 +436,18 @@ enum ShortcutStylePreference: String, CaseIterable, MacroPreference {
     }
 }
 
+enum SortHowPreference: String, CaseIterable, MacroPreference {
+    case recentFirst = "0"
+    case alphabetical = "1"
+
+    var localizedString: LocalizedString {
+        switch self {
+            case .recentFirst: return NSLocalizedString("Recent first", comment: "")
```

Comment:
There are a few more steps to do to make the localization work, but I will do them later on top of this PR, so no problem there.